test/support/oauthUtil.js

```diff
@@ -200,14 +199,15 @@ oauthUtil.setup = function(opts) {
   } else if (opts.tokenRenewArgs) {
     promise = authClient.token.renew.apply(this, opts.tokenRenewArgs);
   } else if (opts.autoRenew) {
-    var renewDeferred = Q.defer();
-    authClient.tokenManager.on('renewed', function() {
-      renewDeferred.resolve();
-    });
-    authClient.tokenManager.on('error', function() {
-      renewDeferred.resolve();
+    promise = new Promise(function(resolve) {
```

Comment:
nit: perhaps a more descriptive var name?

Reply:
This is a test helper setup block, it handles a very diverse range of cases. It doesn't seem like there is a more specific name we can use. It is inside a block ```} else if (opts.autoRenew) {``` so this is the autoRenew promise.